edx-platform/pearson-ukl-theme/lms/templates/header/user_dropdown.html

```diff
@@ -19,6 +19,10 @@
 self.real_user = getattr(user, 'real_user', user)
 profile_image_url = get_profile_image_urls_for_user(self.real_user)['medium']
 username = self.real_user.username
+
+if configuration_helpers.get_value('SHOW_PROFILE_NAME_INSTEAD_OF_USERNAME', False):
```

Comment:
`getattr(self.real_user, 'profile', None)` ? @Squirrel18

Reply:
Not necessary. @Jacatove